Documentation/toc.yml

```diff
@@ -1,169 +1,171 @@
-- name: Getting Started with MRTK

```

Comment:
I think you changed the line-ending style here right -was it that it was inconsistent with the rest of the stuff?

Reply:
I'm not sure - it was part of a mostly-automated merge to resolve a merge conflict.